pom.xml

```diff
@@ -665,6 +665,22 @@
                     </execution>
                 </executions>
             </plugin>
+            
+            <plugin>
+	            <groupId>org.owasp</groupId>
+	                <artifactId>dependency-check-maven</artifactId>
```

Comment:
Please fix the indentation of this `<plugin>` block and add a `<version>` tag for the dependency-check-maven plugin.